File: java/HelloCv.java
import org.opencv.core.CvType;
import org.opencv.core.Mat;
import origami.Origami;

public class HelloCv {
    public static void main(String[] args) throws Exception {
        Origami.init();
        Mat hello = Mat.eye(3, 3, CvType.CV_8UC1);
        System.out.println(hello.dump());
    }
}

File: kotlin/sample/HelloCv.kt
package sample

import org.opencv.core.Core
import org.opencv.core.CvType
import org.opencv.core.Mat
import org.scijava.nativelib.NativeLoader
import origami.Origami

object HelloCv {

    @Throws(Exception::class)
    @JvmStatic
    fun main(args: Array<String>) {
        Origami.init()
        val hello = Mat.eye(3, 3, CvType.CV_8UC1)
        println(hello.dump())
    }
}
